.gitignore

```diff
@@ -5,6 +5,7 @@
 tmp/
 cache/
 output/
+tags
```

Comment:
Is this a file or a directory? Is it placed in the repository root (then it should be `/tags`) or in some other static location?

Reply:
Ah, of course, silly me. Thank you for helping me get it right.